scripts/validate

```diff
@@ -3,18 +3,37 @@ set -e
 
 cd $(dirname $0)/..
 
-echo Running validation
+echo "!! Running validation for go and yaml files"
 
-PACKAGES="$(go list ./...)"
+## validation : go file with golint
+echo "!! Running: go vet"
 
-echo Running: go vet
+PACKAGES="$(go list ./...)"
```

Comment:
If not all dependencies are downloaded you may get additional lines with `go: downloading...`.

```suggestion
PACKAGES="$(go list ./... | grep 'rancher/security-scan')"
```